clients/js/test/mpl_core/deposit.test.ts

```diff
@@ -0,0 +1,444 @@
+import { appendTransactionMessageInstruction, pipe } from '@solana/web3.js';
+import {
+  createAsset,
+  createDefaultAsset,
+  PluginAuthorityPairArgs,
+  VerifiedCreatorsArgs,
+} from '@tensor-foundation/mpl-core';
+import {
+  createDefaultSolanaClient,
+  createDefaultTransaction,
+  expectCustomError,
+  generateKeyPairSignerWithSol,
+  signAndSendTransaction,
+} from '@tensor-foundation/test-helpers';
+import {
+  fetchWhitelistV2,
+  getUpdateWhitelistV2Instruction,
+  Mode,
+  operation,
+  TENSOR_WHITELIST_ERROR__FAILED_FVC_VERIFICATION,
+  TENSOR_WHITELIST_ERROR__FAILED_MERKLE_PROOF_VERIFICATION,
+  TENSOR_WHITELIST_ERROR__FAILED_VOC_VERIFICATION,
+} from '@tensor-foundation/whitelist';
+import test from 'ava';
+import {
+  CurveType,
+  fetchPool,
+  findNftDepositReceiptPda,
+  getDepositNftCoreInstructionAsync,
+  PoolConfig,
+  PoolType,
+} from '../../src/index.js';
+import { createPool, createWhitelistV2 } from '../_common.js';
+
+test('deposit non-whitelisted NFT fails', async (t) => {
```

Comment:
similar to prior deposit files: first test should probably be "deposit whitelisted NFT succeeds"

Reply:
Added